api/ULongRange.json

```diff
@@ -1,53 +0,0 @@
-{
-  "api": {
-    "ULongRange": {
```

Comment:
This is like `ConstrainULong`, a long vs. unsigned long difference in spec and implementations. I'd break these two out into a separate PR.